src/elm/Page/Community/Settings/Features.elm

```diff
@@ -0,0 +1,139 @@
+module Page.Community.Settings.Features exposing (Model, Msg, init, msgToString, update, view)
+
+import Api.Graphql
+import Community exposing (Community)
+import Eos exposing (Symbol)
+import Graphql.Http
+import Html exposing (Html, div, input, label, span, text)
+import Html.Attributes exposing (checked, class, for, id, name, style, type_)
+import Html.Events exposing (onCheck)
+import Page
+import Route
+import Session.LoggedIn as LoggedIn exposing (External(..))
+import UpdateResult as UR
+
+
+init : LoggedIn.Model -> Symbol -> ( Model, Cmd Msg )
+init { shared } symbol =
+    ( initModel symbol
+    , Api.Graphql.query shared (Community.communityQuery symbol) CompletedLoad
+    )
+
+
+initModel : Symbol -> Model
+initModel symbol =
+    { status = Loading
+    , symbol = symbol
+    , actions = False
+    , shop = False
+    , test = ""
+    }
+
+
+type alias Model =
+    { status : Status
+    , symbol : Symbol
+    , actions : Bool
+    , shop : Bool
+    , test : String
+    }
+
+
+type Status
+    = Loading
+    | LoadingFailed (Graphql.Http.Error (Maybe Community))
+    | Loaded Community
+
+
+type Msg
+    = CompletedLoad (Result (Graphql.Http.Error (Maybe Community)) (Maybe Community))
+    | ToggleActions Bool
+    | ToggleShop Bool
+
+
+type alias UpdateResult =
+    UR.UpdateResult Model Msg (External Msg)
+
+
+view : LoggedIn.Model -> Model -> Html Msg
+view loggedIn model =
+    div [ class "bg-white flex flex-col items-center" ]
+        [ Page.viewHeader loggedIn "Features" (Route.CommunitySettings model.symbol)
+        , div
+            [ class "container w-full divide-y"
+            ]
+            [ toggleView "Actions" model.actions ToggleActions "actions"
+            , toggleView "Shop" model.shop ToggleShop "shop"
+            ]
+        ]
+
+
+toggleView : String -> Bool -> (Bool -> Msg) -> String -> Html Msg
+toggleView labelText status toggleFunction inputId =
+    let
+        classes =
+            class "flex items-center"
+
+        statusText =
+            if status == True then
+                "Enabled"
+
+            else
+                "Disabled"
```

Comment:
`status` is not a very good name for this variable. It's better to use `isEnabled` so we can write conditionals like this:

```elm
statusText =
    if isEnabled then
        "Enabled"

    else
        "Disabled"
```